src/hope_payment_gateway/api/fsp/serializers.py

```diff
@@ -37,10 +38,18 @@ def get_payload(self, obj: Any) -> Any:
         }
 
 
+class AccountTypeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = AccountType
+        fields = ("key", "label", "unique_fields")
+
+
 class DeliveryMechanismSerializer(PayloadMixin, serializers.ModelSerializer):
+    account_type = AccountTypeSerializer(read_only=True)
```

Comment:
we just need "key" value in this field, not an entire object

Reply:
ok